### PR TITLE
Tasha's Cauldron of Almost Everything

### PR DESCRIPTION
**Pull request by Azzzzzy**
Part 1 of the Tasha's Content which includes:
- Archetypes
- Feats
- Spells
- Items
- Fighting Style
- Invocations
- Metamagics
- Maneuvers

### CODE REVIEW

**Review by kgirtxd — COMMENTED** [review]
Todo:
Extended spell lists

File: supplements/tashas-cauldron-of-everything/optional-class-features.xml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	<info>
+		<name>Optional Class Features</name>
+		<description>The Optional Class Features from Tasha’s Cauldron of Everything</description>
+		<author url="http://dnd.wizards.com/products/tabletop-games/rpg-products/tashas-cauldron-everything">Tasha’s Cauldron of Everything</author>
+		<update version="0.0.1">
+			<file name="optional-class-features.xml" url="https://raw.githubusercontent.com/AuroraLegacy/supplements/tashas-cauldron-of-everything/optional-class-features.xml" />
+		</update>
+	</info>
+  <element name="Optional Class Features" type="Grants" source="Tasha’s Cauldron of Everything" id="ID_GRANTS_OPTIONAL_CLASS_FEATURES" />
+	<element name="Blessed Warrior" type="Class Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_CLASS_FEATURE_FIGHTING_STYLE_BLESSED_WARRIOR">
```
[the remark this answers is not in the document]
Comment (Azzzzzy) [reply supplements/tashas-cauldron-of-everything/optional-class-features.xml:12]: I think its fine we need a file for basically all of the extra stuff anyways.

File: supplements/tashas-cauldron-of-everything/archetypes/sorcerer-aberrant.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
[the remark this answers is not in the document]
Comment (Azzzzzy) [reply supplements/tashas-cauldron-of-everything/archetypes/sorcerer-aberrant.xml:1]: Note to all: The Program doesn't like apostrophes in supports.

File: supplements/tashas-cauldron-of-everything/archetypes/wizard-bladesinging.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+			<ul>
+				<li>You gain a bonus to your AC equal to your Intelligence modifier (minimum of +1).</li>
+				<li>Your walking speed increases by 10 feet.</li>
+				<li>You have advantage on Dexterity (Acrobatics) checks.</li>
+				<li>You gain a bonus to any Concentration saving throw you make to maintain your concentration on a spell. The bonus equals your Intelligence modifier (minimum of +1).</li>
+			</ul>
+			<p>You can use this feature a number of times equal to your proficiency bonus, and you regain all expended uses of it when you finish a long rest.</p>
+		</description>
+		<sheet action="Bonus Action" usage="{{proficiency}}/Long Rest">
+			<description>You can {{arcana}} {{medicine}}start the Bladesong, which lasts for 1 minute. It ends early if you are incapacitated, if you don medium or heavy armor or a shield, or if you use two hands to make an attack with a weapon. You can also dismiss the Bladesong at any time.
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/archetypes/wizard-bladesinging.xml:63]: "You can {{arcana}} {{medicine}}start the Bladesong"

Looks like a copy paste oops

**Review by JayPea00101010 — APPROVED** [review]
Yes

**Pull request merged**